changelog/265.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+runner: add debug runners to wrap our products' debug commands: SimpleDebug, NomadDebug, VaultDebug, and ConsulDebug.
```

Comment:
The changelog note mention Nomad and Consul which I think are cut for scope? We should update this release note to mention the specific additions in the PR.

Reply:
Ahh thanks, I knew that might bee too ambitious of a changelog entry to write :-)